### PR TITLE
SW-1769: Inventory Tooltips

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR adds tooltips to various pre-existing Inventory screens.

Note: The Inventory Details (Add/Edit/Withdraw) Screens are currently in-progress and tooltips for those screens will be included in follow-up work soon.

### CODE REVIEW

File: src/components/Inventory/InventoryTable.tsx [hunk 1]
```diff
@@ -89,7 +109,7 @@ export default function InventoryTable(props: InventoryTableProps): JSX.Element
         <InventoryFilters filters={filters} setFilters={setFilters} organization={organization} />
       </Grid>
 
-      <Grid xs={12} display='flex' paddingLeft={3} paddingTop={1}>
+      <Grid item xs={12} display='flex' paddingLeft={3} paddingTop={1}>
```
Comment (joshbuchea) [inline src/components/Inventory/InventoryTable.tsx:112]: There were some errors in the console for using `Grid` without an `item` prop. Added this to silence the error.

File: src/strings/index.tsx [hunk 2]
```diff
@@ -700,10 +700,16 @@ const strings = new LocalizedStrings({
...
+    TOOLTIP_SCIENTIFIC_NAME:
       'The binomial Latin name (genus + species) currently accepted in the botanical literature.',
-    TOOLTIP_SPECIES_COMMON_NAME:
-      'The name an organism is known by to the general public, rather than its scientific name.',
+    TOOLTIP_COMMON_NAME: 'The name an organism is known by to the general public, rather than its scientific name.',
+    TOOLTIP_GERMINATING_QUANTITY:
+      'Germinating indicates that seedlings have not yet emerged from seeds. Germinating quantity does not count toward the total quantity.',
+    TOOLTIP_NOT_READY_QUANTITY:
+      'What determines a plant as “not ready” is up to your team and is often species specific. For example: size of plant or amount of time acclimated.',
+    TOOLTIP_READY_QUANTITY:
```
Comment (karthikbtf) [inline src/strings/index.tsx:710]: there are 2 existing tooltips
```
index.tsx:    NOT_READY_QUANTITY_TOOLTIP: 'What determines a plant as “not ready” is up to your team.',
index.tsx:    READY_QUANTITY_TOOLTIP: 'What determines a plant as “ready” is up to your team.',
```
which are used in the inventory create modal.
wondering if those need to be switched out and the new ones added?

File: src/components/Inventory/view/InventorySeedlingsTable.tsx [hunk 3]
```diff
@@ -210,7 +210,7 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
           </Box>
           <InventoryFilters filters={filters} setFilters={setFilters} organization={organization} />
         </Box>
-        <Grid xs={12} display='flex' paddingTop={1}>
+        <Grid item xs={12} display='flex' paddingTop={1}>
```
Comment (joshbuchea) [inline src/components/Inventory/view/InventorySeedlingsTable.tsx:213]: Same here

**Review by karthikbtf — APPROVED** [review]
Looks great 🙌 
one comment/question but otherwise good to merge

File: src/components/Inventory/InventoryTable.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/Inventory/InventoryTable.tsx:112]: i believe those console may be tolerable for now - adding an `item` may need a visual check to see if styles need to be updated.
cc'd you on a closed PR (https://github.com/terraware/terraware-web/pull/614)
i have a jira open to look-into/cleanup console errors. https://terraformation.atlassian.net/browse/SW-1819
if in this particular case adding `item` didn't change any of the styling, we should be super fine with the change.

File: src/strings/index.tsx (hunk 2 above)
Comment (joshbuchea) [reply src/strings/index.tsx:710]: Nice catch, thanks! 🙌  Updated ✅

File: src/components/Inventory/InventoryTable.tsx (hunk 1 above)
Comment (joshbuchea) [reply src/components/Inventory/InventoryTable.tsx:112]: Got it — Going forward I'll make sure to check if visual changes are needed any time an `item` prop is added.

**Review by karthikbtf — APPROVED** [review]
👍

**Review by constanzauanini — APPROVED** [review]

**Pull request merged**